### ISSUE TITLE
Make sure the selected ZIP file is a valid Artemis plugin

### ISSUE THREAD

**Issue by hardtekpt** [IceBotYT/Artemis.Plugins.Modules.Chrome#5]
Hi, I have Artemis installed and would like to get data from chrome. I have installed the browser extension but I have not been able to install the Artemis plugin. When downloading this repository and trying to import it via Settings->Plugins->import plugin, an error pops up saying that the plugins main entry has not been found.

I noticed that in fact there is a .csproj file instead of .DLL file inside the plugin but I have no idea if this is the problem or not. Is it possible for you to provide installation instructions for this plugin?

**Comment by IceBotYT** [comment]
Of course! The download link for the right file is right here 😃

![image](https://github.com/Artemis-RGB/Artemis.Plugins.Modules.Chrome/assets/34712694/aa949cba-f8d0-4818-b774-fd77e565bbc1)

**Comment by hardtekpt** [comment]
Thank you for your fast reply. I cant believe I didnt see that. Still it looks like smth is not right.

![error](https://github.com/Artemis-RGB/Artemis.Plugins.Modules.Chrome/assets/25752785/00e066f0-e502-4a80-b8c2-22508392214d)

**Comment by hardtekpt** [comment]
I have managed to install the plugin. I took a look at this file to see how to build the plugin ([test.yaml](https://github.com/Artemis-RGB/Artemis.Plugins.Modules.Chrome/blob/main/.github/workflows/test.yml)). I installed .NET version 7 and ran this command inside the plugin folder:

dotnet publish -c Release src/Artemis.Plugins.Modules.Chrome.sln

After that, inside the Artemis.Plugins.Modules.Chrome-main\Artemis.Plugins.Modules.Chrome-main\src\Artemis.Plugins.Modules.Chrome\bin\Release\net7.0\publish folder I zipped all the files inside and used that to install the plugin.
